Common/src/main/java/me/egg82/tfaplus/services/InternalAPI.java

```diff
@@ -527,79 +546,76 @@ public boolean seekHOTPCounter(UUID uuid, String[] tokens, JedisPool redisPool,
 
         // SQL
         HOTPData result = null;
-        try {
-            if (sqlType == SQLType.MySQL) {
-                result = MySQL.updateHOTP(sql, storageConfigNode, uuid, data.getLength(), counter, data.getKey()).get();
-            } else if (sqlType == SQLType.SQLite) {
-                result = SQLite.updateHOTP(sql, storageConfigNode, uuid, data.getLength(), counter, data.getKey()).get();
-            }
-        } catch (ExecutionException ex) {
-            logger.error(ex.getMessage(), ex);
-        } catch (InterruptedException ex) {
-            logger.error(ex.getMessage(), ex);
-            Thread.currentThread().interrupt();
-        }
+        result = getHotpData(uuid, data.getLength(), counter, data.getKey(), result);
```

Comment:
Can just be `HOTPData result = getHotpData(uuid, data.getLength(), counter, data.getKey());`
`result` param is redundant.